frappe/social/doctype/energy_point_log/energy_point_log.py

```diff
@@ -37,20 +37,20 @@ def get_alert_dict(doc):
 	})
 	owner_name = get_fullname(doc.owner)
 	doc_link = frappe.get_desk_link(doc.reference_doctype, doc.reference_name)
-	points = frappe.bold(doc.points)
+	points = doc.points
```

Comment:
Call the `get_points_text` function once here, and use that everywhere, since you're not changing the variable.